plugins/utils/makeFeedHtml.js

```diff
@@ -0,0 +1,53 @@
+const { imageWrapperClass } = require('gatsby-remark-images/constants')
+const unified = require('unified')
+const { convertHastToHtml } = require('./convertHast.js')
+const { select, selectAll } = require('hast-util-select')
+
+const rootToAbsolute = siteUrl => tree => {
+  selectAll('a', tree).forEach(node => {
```

Comment:
let's put a link here https://github.com/gatsbyjs/gatsby/issues/14133 to remove this when it's fixed upstream?

Reply:
Sounds good to me!